check/build-changed-protos

```diff
@@ -55,27 +55,9 @@ else
 fi
 
 # All the protos.
-protos=$(find -type f -name "*.proto" -printf '%P\n' \
-    | grep "google/api/.*\.proto$" \
-    | sed -e "s/\.proto$//")
+echo "Building protos in $(pwd)"
 
-# Get the changed protos and strip them of ".proto"
-changed=$(git diff --name-only ${rev} -- \
-    | grep "google/api/.*\.proto$" \
-    | sed -e "s/\.proto$//"
-)
-if [[ ! -z "$changed" ]]; then
-    for base in $changed
-    do
-        echo -e "Found changed proto: '${base}.proto'"
-    done
-fi
-
-# We potentially will attempt to build a second time.
-for base in $changed
-do
-    python -m grpc_tools.protoc -I=cirq-google --python_out=cirq-google --mypy_out=cirq-google ${base}.proto
-done
+dev_tools/build-protos.sh
 
 # Filenames with spaces will be ugly (each part will be listed separately)
 # but the error logic will still work.
```

Comment:
I'd suggest some changes in the following code:

- Changing the variable name from `untracked` to `uncommitted`, since we're looking for any uncommitted changes (e.g. diffs between what was checked in and what the proto scripts generates), not just untracked files. I think `git status --porcelain` does what we want, it's just the variable name and error message that are misleading.
- Also, the grep needs to be changed to reflect the correct path; I think it should be `cirq-google` instead of `cirq/google`, since we want to catch any uncommitted changes in the `cirq-google` package.

Reply:
Changed the grep, although there were additional issues: I should use `-E` and `cut` should use the third field. I tested this by manually editing a file.